### PR TITLE
update full node specs

### PR DESCRIPTION
**Pull request by oxarbitrage**
There was a discussion on what specs are needed to run a full node in the telegram.

The most realistic and affordable at the moment seems to be:

> 64GB ram and 640GB higher iops ssd are requirements for full API node as server
Node will use 102GB on disk for db, 30-40G for OS/software and will load 498GB to ram/swap

Thanks to @dls-cipher

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ In order to run a full node with all the account history you need to remove `par
 
 | Default | Full | Minimal  | ElasticSearch 
 | --- | --- | --- | ---
-| 100G HDD, 16G RAM | 200G HDD, 160G RAM | 80G HDD, 4G RAM | 500G SSD, 32G RAM
+| 100G HDD, 16G RAM | 640 SSD, 64G RAM | 80G HDD, 4G RAM | 500G SSD, 32G RAM
```
Comment (pmconrad) [inline README.md]: 640 **G** Also please add a hint that SSD is to be set up as swap space in that case.

File: README.md [hunk 2]
```diff
@@ -80,7 +80,9 @@ In order to run a full node with all the account history you need to remove `par
 
 | Default | Full | Minimal  | ElasticSearch 
 | --- | --- | --- | ---
-| 100G HDD, 16G RAM | 640 SSD, 64G RAM | 80G HDD, 4G RAM | 500G SSD, 32G RAM
+| 100G HDD, 16G RAM | 640G SSD, 64G RAM * | 80G HDD, 4G RAM | 500G SSD, 32G RAM
+
+\* For this setup to work allocate all SSD space left(excluding OS and software) as Swap.
```
Comment (nathanhourt) [inline README.md]: Perhaps "For this setup, allocate at least 500GB of SSD as swap"?

Comment (pmconrad) [reply README.md]: +1

Comment (oxarbitrage) [reply README.md]: changed, thanks.

**Comment by dls-cipher** [comment]
Can some please merge this, I would like to polish update section where git fetch is missing and would like to add swap setup with some tweaks and explanation for it.

Thanks

**Comment by oxarbitrage** [comment]
This branch(`release`) will be merged to `master` 19 of FEB according to current release schedule. 
We should wait for the changes in this pull request to get into `master` then instead of now.

**Pull request merged**